contracts/token/ControlledToken.sol

```diff
@@ -21,14 +21,18 @@ contract ControlledToken is ERC777UpgradeSafe, BaseRelayRecipient {
     controller = _controller;
   }
 
-  function mint(address _user, uint256 _amount, bytes calldata data, bytes calldata operatorData) external virtual onlyController {
+  function controllerMint(address _user, uint256 _amount, bytes calldata data, bytes calldata operatorData) external virtual onlyController {
     _mint(_user, _amount, data, operatorData);
   }
 
-  function burn(address _user, uint256 _amount, bytes calldata data, bytes calldata operatorData) external virtual onlyController {
+  function controllerBurn(address _user, uint256 _amount, bytes calldata data, bytes calldata operatorData) external virtual onlyController {
     _burn(_user, _amount, data, operatorData);
   }
 
+  // function burn(uint256 amount, bytes memory data) public override  {
```

Comment:
✂️

Reply:
ERC777 has a public "burn" method but it cannot be overridden (not marked virtual)